### PR TITLE
add animated favicon

### PR DESCRIPTION
**Pull request by callum**

### CODE REVIEW

File: apps/pr/lib/pr/play_state.ex [hunk 1]
```diff
@@ -36,6 +36,18 @@ defmodule PR.PlayState do
     end
   end
 
+  def debug_toggle_playing() do
```
Comment (errkk) [inline apps/pr/lib/pr/play_state.ex]: Did ya wanna keep this in?

File: apps/pr_web/lib/pr_web/live/favicon_live.ex [hunk 2]
```diff
@@ -0,0 +1,32 @@
+defmodule PRWeb.FaviconLive do
```
Comment (errkk) [inline apps/pr_web/lib/pr_web/live/favicon_live.ex]: nicely modular :D

Comment (callum) [reply apps/pr_web/lib/pr_web/live/favicon_live.ex]: Yeah, it seemed a bit overkill but thought it was best to keep things consistent. I think LiveView has a concept of live components - reckon that's worth looking into?

File: apps/pr/lib/pr/play_state.ex (hunk 1 above)
Comment (callum) [reply apps/pr/lib/pr/play_state.ex]: I guess it's not really needed, just thought it was a handy little utility. Left it here to prompt your thoughts on it

Comment (errkk) [reply apps/pr/lib/pr/play_state.ex]: shall we remove it for now to keep it simple. Might be an idea to consider tho now we're an open source community 😁  to make a module with manual triggerable helper functions in it or something

File: apps/pr_web/lib/pr_web/live/favicon_live.ex (hunk 2 above)
Comment (errkk) [reply apps/pr_web/lib/pr_web/live/favicon_live.ex]: yeh i like it like this. live component would be good to investigate, but let's get this merged

File: apps/pr/lib/pr/play_state.ex (hunk 1 above)
Comment (errkk) [reply apps/pr/lib/pr/play_state.ex]: actually fuckit its got `debug_` at the start

**Pull request merged**